### DISCUSSION TITLE
Ability to use the top search button as a broweser

### DISCUSSION THREAD

**Discussion by SimonZindo** [andrewmcgivery/obsidian-beautitab#45] [Q&A]
I highly appreciate the plugin beautiful. Is there any ability to use the top search bar as a browser rather than having it only search my vault for notes? If it is possible, I would love to be aware of it.

Again thank you for the hard work you have put into it.

**Comment by fleeflonc** [comment]
+1 to this, not sure why the option was made available if there are no options. If I was more knowledgeable I would try to implement it myself.
